test/parallel_api/experimental/for_loop.pass.cpp

```diff
@@ -296,8 +296,8 @@ test_for_loop()
         Sequence<T> in_out(n, Gen<T>());
         Sequence<T> expected = in_out;
 
-        invoke_on_all_policies<>()(test_for_loop_impl(), in_out.begin(), in_out.end(), expected.begin(), expected.end(),
-                                   in_out.size());
+        invoke_on_all_host_policies()(test_for_loop_impl(), in_out.begin(), in_out.end(), expected.begin(),
```

Comment:
Lets add a comment above that we have just the host implementation of `for_loop` and do the test just for the host policies.

Reply:
Fixed